react-state-2/react-state-2_triple-count/src/components/Counter.js

```diff
@@ -1,7 +1,7 @@
 import { useState } from "react";
 
 export default function Counter() {
-  const [count, setCount] = useState(0);
+  const [count, setCount] = useState(3);
```

Comment:
we shouldn’t change the initial state of count state directly here but call setCount((prevCount) => prevCount + 1); three times